wazimap_ng/general/templates/widgets/VariableFilterWidget.html

```diff
@@ -1,7 +1,12 @@
 <div id="variable-permission-filter">
     {{ choice_field }}
 </div>
-<select id="id_indicator" required name="{{ name }}">
+<input type="hidden" name="selected_profile_id" value="{{ profile_id }}">
```

Comment:
We could add a hidden field using Django forms if required

Reply:
removed this field now that we do not need profile id in frontend